reviews/views.py

```diff
@@ -1,3 +1,38 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from reviews.models        import Review
+from django.http           import JsonResponse, request
+from django.views          import View
+from users.models          import User
+from products.models       import Product
+from users.utils           import login_decorator
+
+class ReviewPostingView(View):
+    @login_decorator    
+    def post(self,request):
+        try:
+            data = json.loads(request.body)
+            if data["title"] == "" :
+                return JsonResponse({"MESSAGE":"NULL_REVIEWS"}, status = 400)
+
+            if data["size_rating"] == "" or data["color_rating"] == "" or data["delivery_rating"] == "" :
+                return JsonResponse({"MESSAGE":"NULL_RATING"}, status = 400)
+
+            if data["size_rating"] >= 6 or data["color_rating"] >= 6 or data["delivery_rating"] >= 6 :
+                return JsonResponse({"MESSAGE":"RATING_INPUT_ERROR"}, status = 400)
+
+            user = User.objects.get(id = request.user.id)
+            Review.objects.create( 
+                user            = User.objects.get(id=request.user.id),
+                product         = Product.objects.get(id=data["product_id"]),
```

Comment:
product_id가 존재하는지 유무를 판단하는 로직을 추가 한 후에 아래와 같이 고치면 좋을 것 같아요.
```suggestion
                product_id         = data["product_id"],
```